Addons/properties-templates/naviox.properties

```diff
@@ -6,6 +6,13 @@ storePasswordAsHex=true
 initialPasswordForAdmin=admin
 # There are many more password options in Configuration module of XavaPro (http://www.openxava.org/xavapro)
 
+#Amazon s3 details
+bucketName=bucketName
+accessKey=accessKey
+secretKey=secretKey
+region=region
+bucketPrefixFolderName=bucketPrefixFolderName
+
 # Navigation options
 startInLastVisitedModule=true

```

Comment:
We need to check why ^M character is appearing in this file. Perhaps because the file is using windows OS file encoding?